src/app/app.coffee

```diff
@@ -4,18 +4,27 @@
 # Doubtfire is modularised into many modules, as indicated by the directory
 # tree inside app/
 #
-#
+
+# Require the CSS file explicitly (or it could be defined as an entry-point too).
+require('../assets/styles/app.scss')
+
+# angular-markdown-filter expects window.showdown to exist
+window.showdown = require('showdown')
+# angular-ui-codemirror expects window.CodeMirror
+window.CodeMirror = require('codemirror')
```

Comment:
Should this be moved into `config/vendor-dependencies/vendor-dependencies.coffee`?